pkg/compiler/codegen.go

```diff
@@ -662,6 +662,7 @@ func (c *codegen) Visit(node ast.Node) ast.Visitor {
 		return nil
 
 	case *ast.AssignStmt:
+		c.saveSequencePoint(n)
```

Comment:
We need a lot of tests for it. It's easy to place these missing lines of code to every AST node handling, but it's not a solution. What we need is to ensure that the resulting sequence points are
1. Properly generated and match the expectations.
2. There are no duplicating sequence points.
3. Match (or at least relatively match) the corresponding C# debugger output for similar AST nodes.